### PR TITLE
[docs] Expand the EAS Metadata documentation

### PR DESCRIPTION
**Pull request by byCedric**
# Why

Fixes ENG-6005

This adds the following pages to the docs:
- Store config documentation - `store.config.json` vs `store.config.js` + `() => ({})` vs `async () => ({})`.
- Store config schema - All configurable options for EAS Metadata
- FAQ - with an pitch / anti-pitch (additional points for this page are welcome)

# How

- Added pages listed above
- Created table helper to quickly generate large tables with data from the [metadata schema](https://github.com/expo/eas-cli/blob/main/packages/eas-cli/schema/metadata-0.json)
- Moved the pages from `/eas-metadata` to `/eas/metadata` + redirects for people accessing old links.

# Test Plan

Start the dev server, and open:
- http://localhost:3002/eas/metadata/
- http://localhost:3002/eas/metadata/getting-started/
- http://localhost:3002/eas/metadata/config/
- http://localhost:3002/eas/metadata/schema/
- http://localhost:3002/eas/metadata/faq/

## Screenshots

<details><summary><b>/eas/metadata</b></summary>

localhost_3002_eas_metadata_

</details>

<details><summary><b>/eas/metadata/getting-started</b></summary>

localhost_3002_eas_metadata_ (1)

</details>

<details><summary><b>/eas/metadata/config</b></summary>

localhost_3002_eas_metadata_ (2)

</details>

<details><summary><b>/eas/metadata/schema</b></summary>

localhost_3002_eas_metadata_ (3)

</details>

<details><summary><b>/eas/metadata/faq</b></summary>

localhost_3002_

</details>

# Checklist

- [ ] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [ ] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [ ] This diff will work correctly for `expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: docs/constants/navigation-deprecated.cjs [hunk 1]
```diff
@@ -12,15 +12,7 @@ const PAGES_DIR = path.resolve(__dirname, '../pages');
...
 
 /** Manual list of directories to categorize as "EAS content" */
-const easDirectories = [
-  'eas',
-  'build',
-  'app-signing',
-  'build-reference',
-  'submit',
-  'eas-update',
-  'eas-metadata',
```
Comment (byCedric) [inline docs/constants/navigation-deprecated.cjs:22]: This now falls under the first entry `eas`.

File: docs/constants/navigation.cjs [hunk 2]
```diff
@@ -12,15 +12,7 @@ const PAGES_DIR = path.resolve(__dirname, '../pages');
...
 
 /** Manual list of directories to categorize as "EAS content" */
-const easDirectories = [
-  'eas',
-  'build',
-  'app-signing',
-  'build-reference',
-  'submit',
-  'eas-update',
-  'eas-metadata',
```
Comment (byCedric) [inline docs/constants/navigation.cjs:22]: This now falls under the first entry eas.

Comment (byCedric) [inline docs/constants/navigation.cjs:22]: This now falls under the first entry eas.

File: docs/pages/eas/metadata/config.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
+---
+title: Create config files for EAS Metadata
```
Comment (EvanBacon) [inline docs/pages/eas/metadata/config.md:2]:
```suggestion
title: Create config files for EAS Metadata
```
This is a bit wordsy, maybe 'Configuring EAS Metadata' like we do with app.json

File: docs/pages/eas/metadata/config.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+import { Callout } from '~/ui/components/Callout';
+import { CodeBlocksTable } from '~/components/plugins/CodeBlocksTable';
+
+<Callout type="warning">
+  EAS Metadata is in beta and subject to breaking changes.
+</Callout>
+<br />
+
+EAS Metadata is designed to be as simple as possible while still being extensible to cover almost all use cases.
+By default, it uses a simple **store.config.json** file located at the _root of your project_.
```
Comment (EvanBacon) [inline docs/pages/eas/metadata/config.md:15]:
```suggestion
EAS Metadata is configured by a **store.config.json** file in the _root of your project_.
```

File: docs/pages/eas/metadata/config.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+```
+
+</CodeBlocksTable>
+
+
+## Dynamic store config
+
+The static config can get you far, but sometimes it's better to use a dynamic value.
+For example, copyright notices usually contain the current year, which is easy to forget to update every year.
+You can make this dynamic by using a **store.config.js** file configured with the [`metadataPath`](../../submit/eas-json.md#metadatapath) property in your **eas.json** file.
```
Comment (EvanBacon) [inline docs/pages/eas/metadata/config.md:57]:
```suggestion
Metadata like the **copyright notice** should contain the current year, with EAS Metadata this can be automated.

Generate content dynamically by creating a JavaScript config file **store.config.js**. Use the [`metadataPath`](../../submit/eas-json.md#metadatapath) property in the **eas.json** file to pick the JS config file.
```

File: docs/pages/eas/metadata/config.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+    }
+  }
+}
+```
+
+</CodeBlocksTable>
+
+## Store config with external content
+
+When using external services for localizations or concepts like "remote config", you have to fetch external content.
```
Comment (EvanBacon) [inline docs/pages/eas/metadata/config.md:91]: What is "remote config"

File: docs/pages/eas/metadata/config.md [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+
+```js
+// Use the data from `eas metadata:pull`
+const config = require('./store.config.json');
+
+const year = new Date().getFullYear();
+
+module.exports = {
+  ...config,
+  copyright: `${year} Acme, Inc.`,
```
Comment (EvanBacon) [inline docs/pages/eas/metadata/config.md:71]: Doesn't the copyright need to be in the `apple` object?

File: docs/pages/eas/metadata/index.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+import { BoxLink } from '~/ui/components/BoxLink';
+import { Callout } from '~/ui/components/Callout';
+import { Terminal } from '~/ui/components/Snippet';
+
+<Callout type="warning">
+  EAS Metadata is in beta and subject to breaking changes.
+</Callout>
+<br />
+
+**EAS Metadata** is a tool that helps submit app store information to publish your app.
```
Comment (EvanBacon) [inline docs/pages/eas/metadata/index.md:16]:
```suggestion
**EAS Metadata** enables you to automate and maintain your app store presence from the command line.
```

File: docs/pages/eas/metadata/index.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+
+You need to provide a lot of information to multiple app stores before your users can use your app. This information is often about complex topics that don't apply to your app. You have to start a lengthy review process after providing the information. When the reviewer finds any issues in the information you provided, you need to restart this process.
+
+## Store config
+
+EAS Metadata aims to make this easier by using a [**store.config.json**](./config.md#static-store-config) file. You can provide all information in this store instead of going through multiple forms in the app store dashboards. When it's time to update the app stores, you can push the store config to the app stores.
```
Comment (EvanBacon) [inline docs/pages/eas/metadata/index.md:22]: This header is strange because it punctuates the previous paragraph, then the next paragraph continues by saying "make this easier".

Comment (EvanBacon) [reply docs/pages/eas/metadata/index.md:22]: Don't say "aims to make this easier", the feature does or doesn't do things.

File: docs/pages/eas/metadata/index.md [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+
+<Terminal cmd={['$ eas metadata:push']} />
+
+## Built-in validation
+
+EAS Metadata does not only get the information into the stores; it also tries to find common pitfalls that could cause app rejections. It does all this before sending anything to the app stores, saving you valuable time to spend on your project.
```
Comment (EvanBacon) [inline docs/pages/eas/metadata/index.md:28]:
```suggestion
EAS Metadata can also instantly identify known app store restrictions that could trigger a rejection after a length review queue.
```

File: docs/pages/eas/metadata/index.md [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+
+You need to provide a lot of information to multiple app stores before your users can use your app. This information is often about complex topics that don't apply to your app. You have to start a lengthy review process after providing the information. When the reviewer finds any issues in the information you provided, you need to restart this process.
+
+## Store config
+
+EAS Metadata aims to make this easier by using a [**store.config.json**](./config.md#static-store-config) file. You can provide all information in this store instead of going through multiple forms in the app store dashboards. When it's time to update the app stores, you can push the store config to the app stores.
+
+<Terminal cmd={['$ eas metadata:push']} />
+
+## Built-in validation
```
Comment (EvanBacon) [inline docs/pages/eas/metadata/index.md:26]: I'd drop this header too

File: docs/pages/eas/metadata/index.md (hunk 10 above)
Comment (EvanBacon) [reply docs/pages/eas/metadata/index.md:28]: specifically, dropping the "tries to find".

File: docs/pages/workflow/glossary-of-terms.md [hunk 12]
```diff
@@ -111,7 +111,7 @@ The `eas.json` file used to configure [EAS CLI](#eas-cli). For more information,
 
 ### EAS Metadata
 
-A command line tool for uploading and downloading Apple App Store metadata as JSON. This tool is available in the `eas-cli` package and should be used to improve the iOS submission process.
+A command line tool for uploading and downloading Apple App Store metadata as JSON. This tool is available in the [EAS CLI](#eas-cli) package and should be used to improve the iOS submission process, see [EAS Metadata](../eas/metadata/index.md).
```
Comment (EvanBacon) [inline docs/pages/workflow/glossary-of-terms.md:114]:
```suggestion
A command line tool for uploading and downloading Apple App Store metadata as JSON. This tool is available in the [EAS CLI](#eas-cli) package and should be used to improve the iOS submission process, see [EAS Metadata](/eas/metadata/index).
```

File: docs/pages/eas/metadata/schema.md [hunk 13]
```diff
@@ -0,0 +1,483 @@
...
+  MetadataTypeCell,
+  MetadataDescriptionCell,
+} from '~/components/plugins/EasMetadataTable';
+
+<Callout type="warning">
+  EAS Metadata is in beta and subject to breaking changes.
+</Callout>
+<br />
+
+The store config in EAS Metadata is a large object containing information that otherwise would be provided manually through the app store dashboards.
```
Comment (EvanBacon) [inline docs/pages/eas/metadata/schema.md:24]:
```suggestion
The store config in EAS Metadata contains information that otherwise would be provided manually through the app store dashboards.
```

File: docs/pages/eas/metadata/index.md (hunk 9 above)
Comment (EvanBacon) [reply docs/pages/eas/metadata/index.md:22]: I'd add a part about how you can store your metadata locally in git.

File: docs/pages/eas/metadata/faq.md [hunk 14]
```diff
@@ -0,0 +1,56 @@
...
+title: EAS Metadata - FAQ
+sidebar_title: FAQ
+---
+
+import { Callout } from '~/ui/components/Callout';
+
+<Callout type="warning">
+  EAS Metadata is in beta and subject to breaking changes.
+</Callout>
+
```
Comment (EvanBacon) [inline docs/pages/eas/metadata/faq.md:11]: I like the idea of this being in a FAQ, we should format it like our other FAQ using collapsing sections. Especially if we implement [improved indexing](https://developers.google.com/search/docs/advanced/structured-data/faqpage) in the components.

File: docs/pages/eas/metadata/faq.md [hunk 15]
```diff
@@ -0,0 +1,56 @@
...
+This dynamic store config allows you to gather information from other places like external services.
+With asynchronous functions, there are no limits to adapting EAS Metadata to suit your preferred workflow.
+
+## Anti-pitch
+
+Here are some reasons EAS Metadata might **not** be the right fit for a project.
+
+### Need features not yet implemented
+
+We are committed to EAS Metadata and will expand functionality over time.
```
Comment (EvanBacon) [inline docs/pages/eas/metadata/faq.md:48]: Rephrase this to be like "Does EAS Metadata support the Google Play Store?" and "How do I use brand new App Store features with EAS Metadata"

File: docs/pages/eas/metadata/config.md (hunk 3 above)
Comment (byCedric) [reply docs/pages/eas/metadata/config.md:2]:
```suggestion
title: Configuring EAS Metadata
```

File: docs/pages/eas/metadata/config.md (hunk 7 above)
Comment (byCedric) [reply docs/pages/eas/metadata/config.md:71]: oopsy

I've looked at this for way too long. Will fix it!

File: docs/pages/eas/metadata/config.md (hunk 6 above)
Comment (byCedric) [inline docs/pages/eas/metadata/config.md:91]:
```suggestion
When using external services for localizations, you have to fetch external content.
```

File: docs/pages/workflow/glossary-of-terms.md (hunk 12 above)
Comment (byCedric) [reply docs/pages/workflow/glossary-of-terms.md:114]: I'm not sure if I follow the suggestion, by using file references we get:
- vscode autocompletion
- validatable links and headers

for example: 
image

File: docs/pages/eas/metadata/faq.md (hunk 14 above)
Comment (byCedric) [reply docs/pages/eas/metadata/faq.md:11]: cc @Simek

File: docs/pages/workflow/glossary-of-terms.md (hunk 12 above)
Comment (Simek) [reply docs/pages/workflow/glossary-of-terms.md:114]: Relative paths are pain when moving the content around, so we stick to the full path.

Removing `.md` is future-proofing, this will bite use when we rename all the files to `.mdx`.

Can you explain a part about links validation? I think it works anyway (if you have the Next validation in mind), but maybe you are talking about remark script?

File: docs/pages/eas/metadata/config.md (hunk 6 above)
Comment (EvanBacon) [reply docs/pages/eas/metadata/config.md]: Maybe we can go with "data fetching" as this is a more common term in web dev and is pretty self-explanatory. Also remote config is generally for A/B testing.

File: docs/pages/workflow/glossary-of-terms.md (hunk 12 above)
Comment (EvanBacon) [reply docs/pages/workflow/glossary-of-terms.md:114]: @byCedric I find it's easy to do this by opening the `docs/pages` folder directly in vs code, then all the auto-complete works as expected.

Comment (byCedric) [reply docs/pages/workflow/glossary-of-terms.md:114]: Next has a lot of configurable options, defined in `next.config.js`. That's on top of the page routing, all of which Markdown itself is unaware of.

This makes validating links pretty hard without Next. Yes, we can spin up the next server, but that would be slow compared to resolving based on the filepath. (and ofc, you get a better workflow in vscode)

Comment (Simek) [reply docs/pages/workflow/glossary-of-terms.md:114]: Still not sure about what exactly validation process of link you are talking about, can you link me a file?

The problem is that even with `.md` and relative paths we still ship broken links to PROD, so I'm not sure if this validation process is working right now, see:
* https://github.com/expo/expo/pull/18895

Additionally, we have committed to correcting those links a good while ago, and currently we are at place where our content is vastly mixed when it comes to the links, also all new content crated do not include the extension in links, so adding `.md` back to the links might be a quite large task.

Maybe we should find or develop new links validation method, which is agnosting when it comes to the form in which links are defined in MD files?

Comment (byCedric) [reply docs/pages/workflow/glossary-of-terms.md:114]: > The problem is that even with .md and relative paths we still ship broken links to PROD, so I'm not sure if this validation process is working right now, see:
It's the [`lint-links` command](https://github.com/expo/expo/blob/main/docs/package.json#L17). Because we can statically evaluate these links, we can validate them.

You are right that we still ship broken links to production, I turned it off because we have a lot of issues already + it takes a long time to run it in CI.
image

I'm open to other solutions though, I think we just lose some benefits if we go for url-based linking.

File: docs/pages/eas/metadata/config.md [hunk 16]
```diff
@@ -0,0 +1,128 @@
+---
+title: Create config files for EAS Metadata
+sidebar_title: Creating the store config
```
[the remark this answers is not in the document]
Comment (EvanBacon) [reply docs/pages/eas/metadata/config.md:3]: "Store config" seems a little bit more ambiguous since it's more of a "Metadata config" (also seems ok). One thing I like about `store.config.json` is that we always use it for downloading, and it implies this is a JSON-related data structure.

[the remark this answers is not in the document]
Comment (byCedric) [reply docs/pages/eas/metadata/config.md:3]: I changed all references to the `store.config.json` to be the term "store config", since we also support 2 different types of "store configs"; `store.config.json` and `store.config.js`. It's just a consistency thing, we can update that if needed.

For the schema, I think we can name that `Metadata Schema`, since that refers to the object within the store config.

[the remark this answers is not in the document]
Comment (byCedric) [reply docs/pages/eas/metadata/config.md:3]: I'll update the sidebar links with this ^

File: docs/pages/eas/metadata/config.md [hunk 17]
```diff
@@ -0,0 +1,124 @@
...
+
+import { Callout } from '~/ui/components/Callout';
+import { CodeBlocksTable } from '~/components/plugins/CodeBlocksTable';
+
+<Callout type="warning">
+  EAS Metadata is in beta and subject to breaking changes.
+</Callout>
+<br />
+
+EAS Metadata is configured by a **store.config.json** file in the _root of your project_.
```
Comment (amandeepmittal) [inline docs/pages/eas/metadata/config.md:14]:
```suggestion
EAS Metadata is configured by a **store.config.json** file at the _root of your project_.
```

The file location is at some place. That's why we use "at the root of the project".

File: docs/pages/eas/metadata/config.md [hunk 18]
```diff
@@ -0,0 +1,124 @@
...
+
+You can configure the path or name of the store config file with the **eas.json** [`metadataPath`](../../submit/eas-json.md#metadatapath) property.
+Besides the default JSON format, EAS Metadata also supports more dynamic config using JavaScript files.
+
+## Static store config
+
+The default store config type for EAS Metadata is a simple JSON file.
+The example below shows an example store config with basic App Store information written in English (U.S.).
```
Comment (amandeepmittal) [inline docs/pages/eas/metadata/config.md:22]:
```suggestion
The code snippet below shows an example store config with basic App Store information written in English (U.S.).
```

File: docs/pages/eas/metadata/config.md [hunk 19]
```diff
@@ -0,0 +1,124 @@
...
+  }
+}
+```
+
+</CodeBlocksTable>
+
+
+## Dynamic store config
+
+Metadata like the **copyright notice** should contain the current year, with EAS Metadata this can be automated.
```
Comment (amandeepmittal) [inline docs/pages/eas/metadata/config.md:54]:
```suggestion
At times, Metadata fields can benefit from dynamic values. For example, the Metadata **copyright notice** should contain the current year. This can be automated with EAS Metadata.
```

File: docs/pages/eas/metadata/config.md [hunk 20]
```diff
@@ -0,0 +1,124 @@
...
+
+Generate content dynamically by creating a JavaScript config file **store.config.js**. Use the [`metadataPath`](../../submit/eas-json.md#metadatapath) property in the **eas.json** file to pick the JS config file.
```
Comment (amandeepmittal) [inline docs/pages/eas/metadata/config.md:56]:
```suggestion
To generate content dynamically, start by creating a JavaScript config file **store.config.js**. Then, use the [`metadataPath`](../../submit/eas-json.md#metadatapath) property in the **eas.json** file to pick the JS config file.
```

File: docs/pages/eas/metadata/config.md [hunk 21]
```diff
@@ -0,0 +1,124 @@
...
+
+</CodeBlocksTable>
+
+## Store config with external content
+
+When using external services for localizations, you have to fetch external content.
+EAS Metadata supports synchronous and asynchronous functions exported from dynamic store config files.
+The function results are awaited before validating and syncing with the stores.
+
+> The **store.config.js** function is evaluated in Node. If you need special values, like secrets, use environment variables.
```
Comment (amandeepmittal) [inline docs/pages/eas/metadata/config.md:92]:
```suggestion
> The **store.config.js** function is evaluated in Node.js. If you need special values, like secrets, use environment variables.
```

File: docs/pages/eas/metadata/faq.md [hunk 22]
```diff
@@ -0,0 +1,64 @@
...
+
+## Pitch
+
+If you are creating or maintaining an app in the stores, keeping the app store information up to date can be a big task.
+When making small changes like supporting a new app store region, you have to find and fill in the proper forms in the app store dashboard.
+After providing all the required information, that content still needs to be approved, where one word can be the difference between approval and rejection.
+
+EAS Metadata aims to make this stage of app development as easy as possible.
+Using a simple configuration file, you can provide all the information to the stores without leaving your project environment.
+Combined with built-in validation, you get instant feedback about the content you provide, even before any review.
```
Comment (amandeepmittal) [inline docs/pages/eas/metadata/faq.md:20]:
```suggestion
If you are creating or maintaining an app in the stores, keeping the app store information up to date can be a big task. When making small changes, for example, supporting a new app store region, you have to find and fill in the forms in an app store's dashboard. Then, after providing all the required information, that content still needs to be approved, where one word can be the difference between approval and rejection.

EAS Metadata aims to make this stage of app development as easy as possible. Using a simple configuration file, you can provide all the information to the stores without leaving your project environment. Combined with built-in validation, you get instant feedback about the content you provide, even before any review.
```

File: docs/pages/eas/metadata/faq.md [hunk 23]
```diff
@@ -0,0 +1,64 @@
...
+
+You can start using EAS Metadata by [creating a new or generating a store config](./getting-started.md#create-the-store-config) from an existing app.
+This store config lets you quickly update the app store information without leaving your project environment.
+Before pushing the changes to the app stores, EAS Metadata looks for common pitfalls that might result in an app rejection.
+
+### Faster feedback loop with validation
+
+EAS Metadata comes with built-in validation, even before anything is sent to the app stores.
+This validation helps you iterate faster over the information without starting a review.
+You can start the review process when everything is provided, and no issues are detected.
```
Comment (amandeepmittal) [inline docs/pages/eas/metadata/faq.md:32]:
```suggestion
EAS Metadata comes with built-in validation, even before anything is sent to the app stores. This validation helps you iterate faster over the information without starting a review. Instead, you can begin the review process when everything is provided, and no issues are detected.
```

File: docs/pages/eas/metadata/faq.md [hunk 24]
```diff
@@ -0,0 +1,64 @@
...
+## Anti-pitch
+
+Here are some reasons EAS Metadata might **not** be the right fit for a project.
+
+### Does EAS Metadata support the Google Play Store?
+
+We are committed to EAS Metadata and will expand functionality over time.
+But that also means that not all functionality is implemented in EAS Metadata.
+The Google Play Store is one of those features currently not implemented.
+See the [store config schema](./schema.md#config-schema) for all existing functionality.
```
Comment (amandeepmittal) [inline docs/pages/eas/metadata/faq.md:51]:
```suggestion
This also means that not all functionality is implemented in EAS Metadata.
The Google Play Store is one of those features currently not implemented.
See the [store config schema](./schema.md#config-schema) for all existing functionality.
```

File: docs/pages/eas/metadata/getting-started.md [hunk 25]
```diff
@@ -0,0 +1,83 @@
...
+        "privacyPolicyUrl": "https://example.com/en/privacy"
+      }
+    }
+  }
+}
+```
+
+</CodeBlocksTable>
+
+> By default, EAS Metadata uses the **store.config.json** file in the root of your project. You can change the name and location of this file by setting the **eas.json** [`metadataPath`](../../submit/eas-json.md#metadatapath) property.
```
Comment (amandeepmittal) [inline docs/pages/eas/metadata/getting-started.md:57]:
```suggestion
> By default, EAS Metadata uses the **store.config.json** file at the root of your project. You can change the name and location of this file by setting the **eas.json** [`metadataPath`](../../submit/eas-json.md#metadatapath) property.
```

File: docs/pages/eas/metadata/getting-started.md [hunk 26]
```diff
@@ -0,0 +1,83 @@
...
+
+## Update the store config
+
+Now it's time to edit the **store.config.json** file and customize it to your app needs. You can find all available options in the [store config schema](./schema.md).
+
+## Upload a new app version
+
+Before pushing the **store.config.json** to the app stores, you must upload a new binary of your app. [Read more about uploading new binaries to stores](../../submit/introduction.md).
```
Comment (amandeepmittal) [inline docs/pages/eas/metadata/getting-started.md:65]:
```suggestion
Before pushing the **store.config.json** to the app stores, you must upload a new binary of your app. For more information, see [uploading new binaries to stores](../../submit/introduction.md).
```

File: docs/pages/eas/metadata/index.md [hunk 27]
```diff
@@ -0,0 +1,48 @@
...
+
+**EAS Metadata** enables you to automate and maintain your app store presence from the command line.
+
+You need to provide a lot of information to multiple app stores before your users can use your app. This information is often about complex topics that don't apply to your app. You have to start a lengthy review process after providing the information. When the reviewer finds any issues in the information you provided, you need to restart this process.
+
+EAS Metadata uses a [**store.config.json**](./config.md#static-store-config) file to provide information instead of going through multiple forms in the app store dashboards. When it's time to update the app stores, you can push the store config to the app stores.
+
+<Terminal cmd={['$ eas metadata:push']} />
+
+EAS Metadata can also instantly identify known app store restrictions that could trigger a rejection after a length review queue.
```
Comment (amandeepmittal) [inline docs/pages/eas/metadata/index.md:24]:
```suggestion
EAS Metadata can also instantly identify known app store restrictions that could trigger a rejection after a lengthy review queue.
```

File: docs/pages/eas/metadata/index.md [hunk 28]
```diff
@@ -0,0 +1,48 @@
...
+
+When adding the store config file to your repository, you can collaborate with others to prepare the app submission.
```
Comment (amandeepmittal) [inline docs/pages/eas/metadata/index.md:26]:
```suggestion
Adding the store config file to your repository enables you to collaborate with other team members to prepare the app submission.
```

Does the changed context fit here? It wasn't clear to me before how just by adding the store config I can collaborate with others to prepare the app submission?

**Review by amandeepmittal — CHANGES_REQUESTED** [review]
Left some suggestions.

File: docs/pages/eas/metadata/index.md (hunk 28 above)
Comment (byCedric) [reply docs/pages/eas/metadata/index.md:26]: In this context, I assumed that you know what a git repo is. The collaboration comes from the part that you can add a file to git + work on it together, as opposed to 1 person going to the app dashboard and filling out all forms. If that makes sense 😄

File: docs/pages/eas/metadata/config.md (hunk 19 above)
Comment (byCedric) [reply docs/pages/eas/metadata/config.md:54]:
```suggestion
At times, Metadata properties can benefit from dynamic values. For example, the Metadata **copyright notice** should contain the current year. This can be automated with EAS Metadata.
```

I think I referred to these as "metadata properties", as in object properties. Will add your suggestion, but with `fields -> properties`, if that's ok with you 😄

File: docs/pages/eas/metadata/index.md (hunk 28 above)
Comment (amandeepmittal) [reply docs/pages/eas/metadata/index.md]: Haha I thought one needed to be part of the organization to collaborate or something like that 😄 (My mind automatically went there).

I still think the changes suggested highlight the advantage a developer gets from it.

**Review by amandeepmittal — APPROVED** [review]
